### PR TITLE
Foam module

### PR DESCRIPTION
**Pull request by fgfuchs**

### CODE REVIEW

File: src/opm/parser/eclipse/EclipseState/Tables/Tables.cpp [hunk 1]
```diff
@@ -482,6 +499,27 @@ const TableColumn& PlymaxTable::getMaxPolymerConcentrationColumn() const {
     return SimpleTable::getColumn(1);
 }
 
+//FoammaxTable::FoammaxTable( const DeckRecord& record ) {
```
Comment (joakim-hove) [inline src/opm/parser/eclipse/EclipseState/Tables/Tables.cpp]: Just remove the commented out code

File: opm/parser/eclipse/EclipseState/Tables/FoamadsTable.hpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/*
+  Copyright (C) 2014 by Andreas Lauser
```
Comment (joakim-hove) [inline opm/parser/eclipse/EclipseState/Tables/FoamadsTable.hpp]: I guess that copyright statement is not entirely correct?

File: src/opm/parser/eclipse/share/keywords/000_Eclipse100/F/FOAMADS [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{"name" : "FOAMADS" , "sections" : ["SPECIAL", "PROPS"], "size" : {"keyword" : "TABDIMS" , "item" : "NTSFUN"} , "items" :
+    [
+        {"name":"DATA", "value_type":"DOUBLE","size_type":"ALL", "dimension":["FoamDensity", "1"]}
```
Comment (joakim-hove) [inline src/opm/parser/eclipse/share/keywords/000_Eclipse100/F/FOAMADS]: Is the string `"FoamDensity"` defined in the unit system parsing code?

**Comment by joakim-hove** [comment]
Hello; 

from a technical opm-common perspective this looks absolute fine, and more or less ready to be merged. When it comes to the larger picture of a foam model in opm-common I am more blank.

Wehn it comes to Work in progress pull requests the tandard approach is to add the work in progress indicator as a leading WIP: in front of the title; I have edited the title to illutstrate.

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/share/keywords/000_Eclipse100/F/FOAMADS (hunk 3 above)
Comment (fgfuchs) [reply src/opm/parser/eclipse/share/keywords/000_Eclipse100/F/FOAMADS]: I assume the changes in src/opm/parser/eclipse/Units/UnitSystem.cpp are what you requested?

File: src/opm/parser/eclipse/EclipseState/Tables/Tables.cpp (hunk 1 above)
Comment (fgfuchs) [reply src/opm/parser/eclipse/EclipseState/Tables/Tables.cpp]: done

File: opm/parser/eclipse/EclipseState/Tables/FoamadsTable.hpp (hunk 2 above)
Comment (fgfuchs) [reply opm/parser/eclipse/EclipseState/Tables/FoamadsTable.hpp]: added copyright statement

**Comment by fgfuchs** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
mind the compiler warnings.

**Comment by atgeirr** [comment]
Replaced by #860.

**Pull request closed without merging**